### PR TITLE
Small things

### PR DESCRIPTION
**Pull request by ncfavier**
# Description

See commits

## Checklist

Before submitting a merge request, please check the items below:

- [X] I've read [the contributing guidelines](https://github.com/plt-amy/1lab/blob/main/CONTRIBUTING.md).
- [X] The imports of new modules have been sorted with `support/sort-imports.hs` (or `nix run --experimental-features nix-command -f . sort-imports`).
- [X] All new code blocks have "agda" as their language.

If your change affects many files without adding substantial content, and
you don't want your name to appear on those pages (for example, treewide
refactorings or reformattings), start the commit message and PR title with `chore:`.

### CODE REVIEW

**Comment by Lavenza** [comment]
[Pull request preview](https://preview.1lab.dev/340)

File: src/Cat/Displayed/Instances/Slice.lagda.md [hunk 1]
```diff
@@ -228,6 +230,33 @@ pullback→cartesian {x} {y} {x'} {y'} {f} {f'} pb = cart where
     pb.unique (sym (m' .commute)) (ap to x)
 ```
 
+The same is true for *weakly* cartesian morphisms, by an equally
+straightforward computation:
+
+```agda
```
Comment (TOTBWF) [inline src/Cat/Displayed/Instances/Slice.lagda.md:236]: Doesn't this follow from https://1lab.dev/Cat.Displayed.Cartesian.Weak.html#4549?

Comment (ncfavier) [reply src/Cat/Displayed/Instances/Slice.lagda.md:236]: It does if you have all pullbacks. I needed this to show that cartesian sections of pullbacks of the codomain fibration are equivalent to equifibered transformations, and there it seems unnecessary to ask for all pullbacks. (And the reason I need the *weakly* cartesian version is that change of base only preserves weakly cartesian morphisms in the direction i'm interested in.)

I concede that this is not very interesting though, so maybe it should be hidden.

Comment (TOTBWF) [reply src/Cat/Displayed/Instances/Slice.lagda.md:236]: That makes sense! I think these proofs are still interesting, though I'd maybe add some prose explaining why these proofs exist so other people don't ask the same question!

**Review by plt-amy — APPROVED** [review]

**Pull request merged**